### PR TITLE
Added build-script for grafana

### PR DESCRIPTION
**Pull request by vinodk99**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: g/grafana/grafana_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#Build Grafana
+mkdir grafana
+cd grafana
+export GOPATH=`pwd`
+git clone https://github.com/grafana/grafana.git
+cd grafana
+git checkout v9.1.3
+
+go mod verify
+yarn run
```
Comment (seth-priya) [inline g/grafana/grafana_ubi_8.5.sh:44]: does this both build and run the tests for the node part?

File: g/grafana/grafana_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+source ~/.nvm/nvm.sh
+nvm install node
+
+#Build Grafana
+mkdir grafana
+cd grafana
+export GOPATH=`pwd`
+git clone https://github.com/grafana/grafana.git
+cd grafana
+git checkout v9.1.3
```
Comment (seth-priya) [inline g/grafana/grafana_ubi_8.5.sh:41]: this shouldn't be hard coded

**Review by seth-priya — CHANGES_REQUESTED** [review]

**Comment by seth-priya** [comment]
https://github.com/grafana/grafana/blob/main/contribute/developer-guide.md this recommends using "yarn install" and "yarn test" for the frontend but we are using "yarn run" here, any specific reason?

File: g/grafana/grafana_ubi_8.5.sh (hunk 2 above)
Comment (vinodk99) [reply g/grafana/grafana_ubi_8.5.sh:41]: Will make changes

File: g/grafana/grafana_ubi_8.5.sh (hunk 1 above)
Comment (vinodk99) [reply g/grafana/grafana_ubi_8.5.sh:44]: This will build the grafana binaries

Comment (seth-priya) [reply g/grafana/grafana_ubi_8.5.sh:44]: ok, refer to https://github.com/grafana/grafana/blob/main/contribute/developer-guide.md  for how to run the tests as well

File: g/grafana/grafana_ubi_8.5.sh (hunk 2 above)
Comment (seth-priya) [reply g/grafana/grafana_ubi_8.5.sh:41]: check and adhere to build script templates, typically we have variables for PACKAGE_NAME, PACKAGE_VERSION, PACKAGE_URL and also provide the flexibilty to the user to pass another version to the build script as an argument

**Pull request closed without merging**